### PR TITLE
Attempt at distributing ownership between AML and APR.

### PR DESCRIPTION
**Pull request by remeh**
### What does this PR do?

An attempt at distributing ownership between AML and APR.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -258,10 +258,10 @@
...
 /comp/core/tagger @DataDog/container-integrations
 /comp/core/workloadmeta @DataDog/container-integrations
-/comp/forwarder/eventplatform @DataDog/agent-metrics-logs
-/comp/forwarder/eventplatformreceiver @DataDog/agent-metrics-logs
-/comp/forwarder/orchestrator @DataDog/agent-metrics-logs
-/comp/forwarder/orchestrator/orchestratorinterface @DataDog/agent-metrics-logs
+/comp/forwarder/eventplatform @DataDog/agent-processing-and-routing
+/comp/forwarder/eventplatformreceiver @DataDog/agent-processing-and-routing
+/comp/forwarder/orchestrator @DataDog/agent-processing-and-routing
+/comp/forwarder/orchestrator/orchestratorinterface @DataDog/agent-processing-and-routing
```
Comment (vickenty) [inline .github/CODEOWNERS:264]: I think these are redundant due to `/comp/forwarder` above.

**Review by vickenty — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (remeh) [reply .github/CODEOWNERS:264]: You're right! Fixing this, thank you

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -404,6 +404,8 @@
 /pkg/logs/                              @DataDog/agent-metrics-logs
 /pkg/logs/launchers/windowsevent        @DataDog/agent-metrics-logs @DataDog/windows-agent
 /pkg/logs/tailers/windowsevent          @DataDog/agent-metrics-logs @DataDog/windows-agent
+/pkg/logs/processor                     @DataDog/agent-processing-and-routing
```
[the remark this answers is not in the document]
Comment (remeh) [reply .github/CODEOWNERS:403]: Oh right, completely forgot the logs forwarding in the original commit!

**Review by gh123man — APPROVED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -239,7 +239,7 @@
 /comp/collector @DataDog/agent-metrics-logs
 /comp/core @DataDog/agent-shared-components
 /comp/dogstatsd @DataDog/agent-metrics-logs
-/comp/forwarder @DataDog/agent-shared-components
+/comp/forwarder @DataDog/agent-processing-and-routing
```
Comment (ogaca-dd) [inline .github/CODEOWNERS:242]: This is automatically generated: https://github.com/DataDog/datadog-agent/pull/24188/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R233

**Review by ogaca-dd — APPROVED** [review]
LGTM

File: .github/CODEOWNERS (hunk 3 above)
Comment (remeh) [reply .github/CODEOWNERS:242]: Oh interesting, can you point me how it's assigned to teams, please?

Comment (ogaca-dd) [reply .github/CODEOWNERS:242]: It is managed by [inv lint-components](https://github.com/DataDog/datadog-agent/pull/24188/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R233) which uses `team` flag (`// team: agent-shared-components`) from component.go files.

**Review by jszwedko — APPROVED** [review]

**Review by CelianR — APPROVED** [review]

**Review by sgnn7 — APPROVED** [review]

**Pull request merged**